Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json

```diff
@@ -26,6 +27,31 @@
                                 "$ref": "#/components/schemas/Node"
                             }
                         }
+                    },
+                    "metadata": {
+                        "type": "object",
+                        "properties": {
+                            "version": {
+                                "$ref" : "#/components/schemas/Version"
+                            }
+                        }
+                    },
+                    "versions": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/VersionPatch"
+                        }
+                    },
+                    "nodeVersionDifferences": {
+                        "description": "A dictionary property that maps from a node identifier to a dictionary of diffs for that node. The node diffs dictionary maps from a version to the type of change (modified/added) the current node should display when diffing against that prior version. If nothing changed against a given version, that version won't appear in the dictionary of diffs.",
```

Comment:
For some reason, adding a `summary` property as well causes [the validator](https://apitools.dev/swagger-parser/online/) to say the spec if invalid.